tests/wpt/tests/css/CSS2/text/text-align-justify-with-overflow.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+
+<head>
+    <title>CSS Test: Overflowing content with text-align: justify</title>
+    <link rel="author" title="Martin Robinson" href="mrobinson@igalila.com">
+    <link rel="help" href="https://drafts.csswg.org/css-text-3/#text-indent-property">
+    <link rel="help" href="http://www.w3.org/TR/CSS21/text.html#indentation-prop">
+    <link ref="help" href="https://drafts.csswg.org/css-text/#text-align-property">
+    <link rel="match" href="text-align-justify-with-overflow-ref.html"/>
+    <style type="text/css">
+        div {
+            text-align: justify;
+            border: solid;
+            width: 100px;
+            white-space: pre-wrap;
+        }
+    </style>
+</head>
+<body>
+
+<!-- These two divs should overflow, one because of the length of the content
+      and the other because of `text-indent`. In both of these cases, `text-align: justify`
+      should be treated like `text-align: left` since there is no extra space to
+      distribute to justification opportunities. -->
+<div style="width: 50px">lorem&nbsp;ipsum</div>
+<div style="width: 100px; text-indent: 50px">lorem&nbsp;ipsum</div>
```

Comment:
You need more text in order to test this, because the last line is not justified (would need `text-align-last`).

Also they should have `width: 0px` to ensure that the text overflows regardless of the font. I just used something bigger because the text was disappearing instead of overlapping (probably a different bug?) so it was harder to notice the problem. Or use Ahem.

```suggestion
<style>div { width: 0 }</style>
<div>lorem&nbsp;ipsum dolor</div>
<div style="text-indent: 50px">lorem&nbsp;ipsum dolor</div>
```

Reply:
I've uploaded a new version of the new test.